docs/UserGuide.md

```diff
@@ -324,12 +325,17 @@ Volunteer Roles (Code - Role Name):
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+| Action                | Format, Examples                                                                               |
+|-----------------------|------------------------------------------------------------------------------------------------|
+| **Add Person**        | `add n/NAME p/PHONE_NUMBER m/EMAIL [r/ROLE]…​`                                                 |
+| **Add Event**         | `addevent sp/SPORT t/Faculty 1 t/Faculty 2 d/LocalDateTime v/Venue [pa/PARTICIPANTS`           |
+| **Clear All Entries** | `clear`                                                                                        |
+| **Delete Person**     | `delete INDEX`                                                                                 |
+| **Delete Event**      | `deleteevent INDEX`                                                                            |
+| **Edit Person**       | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`                         |
```

Comment:
Please take note that a person does not have an address anymore, could we change this?

Reply:
forgot to change this for editing a person specifically, will fix